runtime/include/perf_window.h

```diff
@@ -158,3 +162,31 @@ perf_window_get_count(struct perf_window *self)
 
 	return self->count;
 }
+
+/**
+ * Print the items in the perf window
+ */
+static inline void
+perf_window_print(struct perf_window *self) 
+{
+	assert(self != NULL);
+	if (self->count % PERF_WINDOW_BUFFER_SIZE != 0) { return; }
+	/* Not need to hold lock because this operation won't add/delete the array */
+	float min = self->by_duration[0].execution_time/1000.0;
+	float max = self->by_duration[PERF_WINDOW_BUFFER_SIZE-1].execution_time/1000.0;
+	uint64_t sum = 0;
+        float fifty_p = self->by_duration[PERF_WINDOW_BUFFER_SIZE * 50 / 100].execution_time/1000.0;	
+        float seventy_p = self->by_duration[PERF_WINDOW_BUFFER_SIZE * 70 / 100].execution_time/1000.0;	
+        float eighty_p = self->by_duration[PERF_WINDOW_BUFFER_SIZE * 80 / 100].execution_time/1000.0;	
+        float nighty_p = self->by_duration[PERF_WINDOW_BUFFER_SIZE * 90 / 100].execution_time/1000.0;	
+        float nighty_night_p = self->by_duration[PERF_WINDOW_BUFFER_SIZE * 99 / 100].execution_time/1000.0;	
```

Comment:
*ninety and *ninety_nine ?

Reply:
ah, I guess you suggest to rename the variables here, right? I am sorry the name is very confused, "p" does not mean pointer, it means percentile.